lib/templates/tabMenuLink.mustache

```diff
@@ -0,0 +1,8 @@
+<li class="res-tabmenu-button">
+	<a class="{{ className }}" {{#title}}title="{{ title }}"{{/title}}>
+	<label>
+		<input type="checkbox" {{checked}}>
+		{{text}}
```

Comment:
Fixed. `.tabmenu li` uses `display: inline`, which for some reason caused that extra margin.